### DISCUSSION TITLE
Help needed: Migration v115.x from Synology-Docker to v120.x zu Proxmox-LXC

### DISCUSSION THREAD

**Discussion by morecolor** [immich-app/immich#14157] [Q&A] [answered]
I have immich server v115.x running in a docker environment on a synology.
The database and the images are stored externally on a Synology file share (No. 1).

Since I am currently moving many docker containers to proxmox-lxc, I have installed the immich server v120.x in a proxmox-lxc, copied the entire Synology share (No.1) into a new Synology share (No.2) and adapted for the LXC-installation the docker-compose.yml and the .env to the Synology share (No. 2).

Both Immich-server are running independently of each other.
How do I now see the images from the Synology share (No.2) in the LXC in version v120.x?

Sorry, I'm still new to Immich. Maybe it's really simple...

**Comment by bo0tzz** [comment, marked as answer]
https://immich.app/docs/administration/backup-and-restore

**Reply by morecolor** [reply]
Tnahnks, I have read this howto. But due to a lack of understanding and the language barrier (Google Translate is unfortunately not the golden key for such technical topics), I don't understand what I have to do step by step. I lack the background understanding of the architecture of immich. I assumed that if I set up immmich again in an lxc and reference the paths of the existing files and the database, I would then simply get access to my image data again.
